build.gradle.kts

```diff
@@ -67,33 +67,66 @@ publishing {
 
 ant.importBuild("build.xml") {old ->"ant-${old}"}
 
+// the normal classpath
+val classpathCollection: FileCollection = layout.files(
+    file(project.properties["build.classes"]),
+    fileTree(mapOf("dir" to project.properties["build.lib.dir"], "include" to listOf("*.jar")))
+)
+val classpath: String = classpathCollection.asPath
+
+// ant target "init" renamed to "init-nutch" to avoid gradle naming conflits
+tasks.register<Copy>("init-nutch") {
+    doLast {
+        // making six directories
+        mkdir(project.properties["build.dir"])
+        mkdir(project.properties["build.classes"])
+        mkdir(project.properties["release.dir"])
+        mkdir(project.properties["test.build.dir"])
+        mkdir(project.properties["test.build.classes"])
+        mkdir(project.properties["test.build.lib.dir"])
+
+        // incomplete: changing datetime for all **/*.template in conf.dir
+        fileTree(mapOf("dir" to project.properties["conf.dir"], "include" to listOf("**/*.template"))).forEach{ file: File ->
+            println(file)
+            // FIX: cannot find touch or datetime equivalent
```

Comment:
Please remove I have absolutely no idea what this was for. 
I looked back in the Git history and it looks like it has been there since March 1st 2005 when the Nutch project was originally stored in SVN (predecessor to Git)
https://github.com/apache/nutch/blob/bd718ff3ab1f923593e5eccaa0aaeb21370f7169/build.xml#L39